basic/03d_command_pos_vel.py

```diff
@@ -47,5 +48,30 @@
 
 # Stop logging. `log_file` contains the contents of the file
 log_file = group.stop_log()
-hebi.util.plot_logs(log_file, 'position', figure_spec=101)
```

Comment:
There was a bug in the Python API, which I _believe_ has since been fixed, related to this (first `plot_logs` would work, but second would not, since the log file had been iterated through)... If it's fixed, we can revert the changes on the RHS here back to what it was previously.

Reply:
:+1: this would be great to test and revert if possible :)